handlers/fb_handlers.php

```diff
@@ -24,18 +24,20 @@
 // and Facebook
 
 // TODO: this only works if the script is installed in root
-define('FACEBOOK_SDK_V4_SRC_DIR', __DIR__ . '/../include/facebook-php-sdk-v4/src/Facebook/');
-require_once(__DIR__ . '/../include/facebook-php-sdk-v4/autoload.php');
+define('FACEBOOK_SDK_V4_SRC_DIR', __DIR__ . '/../include/php-graph-sdk/src/Facebook');
+require_once(__DIR__ . '/../include/php-graph-sdk/src/Facebook/autoload.php');
```

Comment:
The autoloader was removed on https://github.com/facebook/php-graph-sdk/commit/877e97749fc45a871a3f522b257bf66491e087e8. Maybe set `branch = 5.4` on .gitmodules fix this.